spec/components/provider_interface/activity_log_event_component_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::ActivityLogEventComponent do
+  def component_for(event)
+    described_class.new(activity_log_event: event)
+  end
+
+  def with_event(trait)
+    event = create(:application_choice_audit, trait)
+    user = event.user.display_name
+    candidate = event.auditable.application_form.full_name
+    yield event, user, candidate
+  end
+
+  describe '#event_description' do
+    examples = {
+      awaiting_provider_decision: '<candidate> submitted an application',
+      withdrawn: '<candidate> withdrew their application',
+      with_rejection: '<user> rejected <candidate>’s application',
+      with_rejection_by_default: '<candidate>’s application was rejected automatically',
```

Comment:
```suggestion
      with_rejection_by_default: '<candidate>’s application was automatically rejected',
```
This is to fit the way we write it in other parts of the service. Putting 'automatically' first places emphasis on it.